recurring_contract/tests/test_recurring_contract.py

```diff
@@ -239,18 +147,16 @@ def test_generated_invoice_second_scenario(self):
 
         # Copy of one contract to test copy method()
         contract2.copy()
-        contract_copied_id = contract_obj.search([
+        contract_copied = self.env['recurring.contract'].search([
```

Comment:
In v8 you can simply do `contract_copied = contract2.copy()`, as the copy method returns the new object.